File: nodes/divide/divide.process.py
async def process(inputs, settings, config, nodeConfig):

  # avoid division by zero 
  if inputs.get("b") == 0:
    return {
      "result": None
    }

  a = float(inputs.get("a", 0))
  b = float(inputs.get("b", 1))  # Avoid division by zero
  
  return {
    "result": a / b
  } 

File: nodes/divide/divide.process.js
export default async ({inputs, settings, config}) => {


  // avoid division by zero 
  if (inputs.b === 0) {
    return {
      result: null
    };
  }

  const a = Number(inputs.a) || 0;
  const b = Number(inputs.b) || 1; // Avoid division by zero
  
  
  return {
    result: a / b
  };
}; 